src/mds/Server.h

```diff
@@ -355,6 +355,7 @@ class Server {
 
   DecayCounter recall_throttle;
   time last_recall_state;
+  bool forward_all_requests_to_auth;
```

Comment:
don't duplicate forward_all_requests_to_auth.  define a funtion in MDCache, to access this

Reply:
I think set_trace_dist is a frequently called function. so I define a variable in Server. I have add a inline func in mdcache.